server/routes/users.js

```diff
@@ -18,7 +18,7 @@ router.get('/', function(req, res, next) {
         return next(err);
     }
     res.send(result.rows);
-  });
```

Comment:
Seems like there was some confusion, but this should be indented to match the corresponding opening brackets.

e.g.
```
{
    {
         {
         }
    }
    {
    }
}
```